public/app/plugins/datasource/cloudwatch/components/LogsQueryField.tsx

```diff
@@ -130,6 +131,12 @@ export class CloudWatchLogsQueryField extends React.PureComponent<CloudWatchLogs
       return Promise.resolve();
     }
 
+    // No need to fetch matching log groups if the search term isn't valid
+    // This is also useful for preventing searches when a user is typing out a log group with template vars
+    if (!/^[\.\-_/#A-Za-z0-9]+$/.test(searchTerm)) {
```

Comment:
Can we add some link to cloudwatch docs where the allowed chars are mentioned for future reference?

Reply:
Sure